scripts/bump_release.sh

```diff
@@ -26,7 +26,11 @@ git checkout release_$NEW_GARMADON_RELEASE || git checkout -b release_$NEW_GARMA
 # Bump release
 mvn versions:set -DnewVersion=${NEW_GARMADON_RELEASE}
 find . -name pom.xml.versionsBackup -delete
-sed -i "s#<garmadon.version>.*</garmadon.version>#<garmadon.version>${NEW_GARMADON_RELEASE}</garmadon.version>#g" pom.xml
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
Already told you that it won't be fine as this depends on sed version (GNU or mac one)
I'm using a mac but used the GNU release of sed, so it won't work on my computer now